### ISSUE TITLE
Pre-training on ImageNet

### ISSUE THREAD

**Issue by dojoscan** [BichenWuUCB/squeezeDet#47]
Hey, I attempted to pre-train the SqueezeDet model on ImageNet, freeze the first conv layer and then train on KITTI but the performance was significantly poorer than that stated in the paper. For ImageNet, I trained for ~50 epochs and the validation accuracy reached ~40% and subsequently trained on KITTI for 100k iterations. At its highest, the KITTI mAP was approximately 60%. I'm just curious as to the conditions you used for pre-training. Thanks!

**Comment by BichenWuUCB** [comment]
The SqueezeNet pretrained model is provided in this repository and the [SqueezeNet repository](https://github.com/DeepScale/SqueezeNet/tree/master/SqueezeNet_v1.1). But if for some reason you need to pretrain it on ImageNet classification from scratch, make sure that the model reached the classification accuracy described the SqueezeNet paper, then finetune it for object detection.

**Comment by dojoscan** [comment]
Ok, thank you for the reply!

**Comment by alex96295** [comment]
@BichenWuUCB Where can I find the pre-trained squeezeNet model? In the repository you indicate here, it is in .caffemodel format, you give the .pkl format in the code, is it the same?
